client/js/components/user/orgaDataEntry/OrganisationCompetence.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <fieldset
+    v-if="hasPermission('field_organisation_competence')"
+    id="organisationCompetence"
+    class="w-3/4">
+    <legend class="font-size-large weight--normal u-mb-0_75">
+      {{ Translator.trans('competence') }}
+    </legend>
+    <div
+      class="w-full mb-3">
+      <dp-text-area
+        id="orga_competence"
+        data-cy="organisationData:paperCopy:competence"
+        :name="`${organisation.id}:competence`"
+        :value="organisation.competence"
```

Comment:
I know it was like this before, but shouldn't we use v-model here? :thinking:

Reply:
I think it doesn't matter in this case, but I changed it here [785572d](https://github.com/demos-europe/demosplan-core/pull/4237/commits/785572dcbbcbd10c664361ed3411ab48bbd8de84)